packages/manager/modules/pci/src/projects/project/storages/cloud-archives/cloud-archive/cloud-archive.routing.js

```diff
@@ -14,6 +26,30 @@ export default /* @ngInject */ ($stateProvider) => {
           containerId,
           objectId: object.name,
         }),
+
+        goBack: /* @ngInject */ goToStorageContainers => goToStorageContainers,
+
+        goToStorageContainer: /* @ngInject */ ($rootScope, CucCloudMessage, $state, projectId, containerId) => (message = false, type = 'success') => {
+          const reload = message && type === 'success';
+
+          const promise = $state.go('pci.projects.project.storages.archives.archive', {
+            projectId,
+            containerId,
+          },
+          {
+            reload,
+          });
+
+          if (message) {
+            promise.then(() => CucCloudMessage[type](message, 'pci.projects.project.storages.containers.container'));
+          }
+
+          return promise;
+        },
+
+        refresh: /* @ngInject */ goToStorageContainer => goToStorageContainer,
+
```

Comment:
Useless whitespace 😄 (I don't care, but you always tell me you will disapprove my PRs because of my useless whitespaces)